README.rst

```diff
@@ -0,0 +1,19 @@
+======
+pyrate
+======
+
+
+Python AIS Tools Environment
+
+
+Description
+===========
+
+A LONGER DESCRIPTION OF YOUR PROJECT GOES HERE...
+
+
+Note
+====
+
+This project has been set up using PyScaffold 2.5.3. For details and usage
+information on PyScaffold see http://pyscaffold.readthedocs.org/.
```

Comment:
Good point - will add a markdown parser to the sphinx script and use the .md readme.